src/bin/CMakeLists.txt

```diff
@@ -272,7 +272,7 @@ configure_file(msquic-config.cmake.in ${CMAKE_BINARY_DIR}/msquic-config.cmake)
 install(FILES ${CMAKE_BINARY_DIR}/msquic-config.cmake DESTINATION share/msquic)
 
 if(BUILD_SHARED_LIBS)
-    install(EXPORT msquic DESTINATION share/msquic)
+    install(EXPORT msquic NAMESPACE msquic:: DESTINATION share/msquic)
```

Comment:
I'm happy seeing CMake namespaces being added to exported targets. But it should be clear that this is a breaking change if not complemented by providing an alias with the old name. I wonder if a breaking change is desired in a patch release.

Reply:
I already submit it to upstream and it accepted. and maybe it will be release.
https://github.com/microsoft/msquic/pull/4692
https://github.com/microsoft/msquic/pull/4701

Reply:
This repo is upstream. That's why I raise the point:
Adding the namespace without add polyfill breaks users.

Reply:
How do we add the alias? I assume this is only a build time breaking change? Regardless, I would like to prevent those, if possible.

Reply:
The namespace is a useful service for consumers only: It signals to CMake that the identifier is a CMake target, not plain input to the linker. This improves detecting configuration errors.

Alias could be setup in `msquic-config.cmake`, after including the generated export files. Sth. like
~~~cmake
foreach(_t IN ITEMS msquic msquic_platform ...)
    if(TARGET msquic::${_t} AND NOT TARGET ${_t})
        add_library(${_t} ALIAS msquic::${_t})
    endif()
endforeach()
~~~

Reply:
This could be removed in 2.5 or 3, after proper deprecation.

Reply:
@dg0yt How you can add this to `msquic-config.cmake`? this file generate automatic. I will glad for your help.